neuromation/cli/job.py

```diff
@@ -928,11 +927,10 @@ def format_fail(job: str, reason: Exception) -> str:
     env_dict = build_env(env, env_file)
 
     if cmd is None:
+        # NOTE: `cmd` is never `None`
```

Comment:
Agree, cmd is never `None`.
If you are touching this piece of code, please remove the branch for `None` entirely.